acloud/resource_cluster.go

```diff
@@ -91,7 +93,9 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, m interf
 	if err != nil {
 		return diag.FromErr(err)
 	}
+
 	if cluster != nil {
+		WaitUntilClusterIsRunning(ctx, m, org, *cluster)
```

Comment:
I think we should do this after setting the state values.

Also, we are not handling any errors from `WaitUntilClusterIsRunning` currently.